HW1/P4/P4.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import division
+from P4_bfs import *
+import numpy as np
+import pyspark
+
+sc = pyspark.SparkContext(appName = "Spark1")
+
+marvelData = sc.textFile('marvel.csv').cache()
+
+marvelList = marvelData.map(lambda x: x.split('"'))
+charL = marvelList.map(lambda x: (x[1],x[3])).partitionBy(10)
```

Comment:
nit: nice work with the partitionBy, although it'd be safer to define a global constant for the number of partitions to make sure RDDs have the same partitioner.